src/app/scripts/cac/routing/cac-routing-itinerary.js

```diff
@@ -0,0 +1,132 @@
+CAC.Routing.Itinerary = (function ($, L, _) {
+
+    /**
+     * Class represents an itinerary between two points
+     *
+     * @param {object} otpItinerary OTP itinerary
+     * @param {integer} index integer to uniquely identify itinerary
+     */
+    function Itinerary(otpItinerary, index) {
+        this.id = index.toString();
+        this.via = getVia(otpItinerary);
+        this.modes = getModes(otpItinerary);
+        this.distanceMiles = getDistanceMiles(otpItinerary);
+        this.durationMinutes = getDurationMinutes(otpItinerary);
+
+        this.geojson = L.geoJson({type: 'FeatureCollection',
+                                  features: getFeatures(otpItinerary.legs)});
+    };
+
+    Itinerary.prototype.getStyle = getStyle;
+    Itinerary.prototype.plotOnMap = plotOnMap;
+
+    return Itinerary;
+
+    /**
+     * Helper function to get label/via summary for an itinerary.
+     * Chooses the streetname with the longest distance for an
+     * itinerary.
+     *
+     * @param {object} otpItinerary OTP itinerary
+     *
+     * @return {string} string to use for labeling an itinerary
+     */
+    function getVia(otpItinerary) {
+        var steps = _(otpItinerary.legs).map(function(leg) {
+            return leg.steps;
+        }).flatten();
+
+        return steps.max(function(step) {
```

Comment:
_.max can be shorthanded to just take a string property name, e.g. `return steps.max('distance')`